bookkeeper-server/src/main/java/org/apache/bookkeeper/server/http/service/MetricsService.java

```diff
@@ -65,6 +67,7 @@ public HttpServiceResponse handle(HttpServiceRequest request) throws Exception {
             statsProvider.writeAllMetrics(writer);
             writer.flush();
             response.setCode(StatusCode.OK);
+            response.setContentType(PROMETHEUS_CONTENT_TYPE_004);
```

Comment:
Use `TextFormat.CONTENT_TYPE_004` instead of defining a new one. Just like: https://github.com/apache/bookkeeper/blob/eb27e6ed3ccd1ee46f372ae6fd0a53ccea81908f/stats/bookkeeper-stats-providers/prometheus-metrics-provider/src/main/java/org/apache/bookkeeper/stats/prometheus/PrometheusServlet.java#L42

Reply:
This will require dependency changes wasn't sure of that. I can either change the scope of the org.apache.bookkeeper.stats:prometheus-metrics-provider artifact to non-test or add an explicit dependency on io.prometheus:simpleclient_common

Reply:
I chose the former approach.